File: FMAplication/Domain/DailyTasks/DailyPosmTask.cs
using System.Collections.Generic;
using FMAplication.Core;
using FMAplication.Domain.Bases;
using FMAplication.Domain.Sales;
using FMAplication.Enumerations;

namespace FMAplication.Domain.DailyTasks
{
    public class DailyPosmTask : DailyBaseTask
    {
        public string ExistingImage { get; set; }
        public string NewImage { get; set; }
        public List<DailyPosmTaskItems> DailyPosmTaskItems { get; set; }
    }
}

File: FMAplication/ClientApp/src/app/Shared/Entity/DailyTasks/dailyPosmTask.ts
import { DailyBaseTask } from "./dailyBaseTask";
import { DailyPosmTaskItem } from "./dailyPosmTaskItem";

export class DailyPosmTask extends DailyBaseTask{
    existingImage: string;
    newImage: string;
    dailyPosmTaskItems: DailyPosmTaskItem[];
}